config/locales/en.yml

```
@@ -967,6 +967,16 @@ en:
       default: Selection from a list
       none_of_the_above: None of the above
       only_one_option: Selection from a list, one option only.
+  page_route_card:
+    check_your_answers: Check your answers before submitting
+    conditional_answer_value: '"%{answer_value}"'
+    conditional_route_title: Route %{index}
+    continue_to: Continue to
```

```suggestion
    continue_to: continue to
```
Lower case cos this is a continuation of a sort of sentence.